### ISSUE TITLE
[HttpFoundation] remaining UploadedFile size deprecation notices

### ISSUE THREAD

**Issue by craue** [symfony/symfony#25466]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 4.1

After #25324, using file form fields (via `Symfony\Component\DomCrawler\Field\FileFormField::upload()` for testing) still triggers deprecation notices like `Passing a size in the constructor is deprecated since 4.1 and will be removed in 5.0. Use getSize() instead`, e.g. from https://github.com/symfony/symfony/blob/28bf816e74b69d7e9161f6163b489090e95148e3/src/Symfony/Component/HttpFoundation/FileBag.php#L87

**Comment by Simperfit** [comment]
This was forgotten I think, I've created a new PR which fix this and add a test to ensure that we have not this behaviour back.
